apps/frontend/src/app/page.tsx

```diff
@@ -1,15 +1,96 @@
+'use client';
+import { Plus } from 'lucide-react';
 import Link from 'next/link';
 
+import { CommentSection } from '@/components/comment-section';
+import { DrinkList } from '@/components/drinkAction-drink-list';
+import { EventDiscription } from '@/components/event-discription';
+import { ShareLink } from '@/components/share-link';
+import { Button } from '@/components/ui/button';
+import {
+  Pagination,
+  PaginationContent,
+  PaginationEllipsis,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+} from '@/components/ui/pagination';
+import { UserAvatar } from '@/components/user-avatar';
+import { UserHoverCard } from '@/components/user-hovercard';
 import { events } from '@/models/mockData';
 
-export default function Home() {
+import { LikeButton } from '../../src/components/like-button';
+
+export default function EventFeedPage() {
   return (
-    <main className='flex items-center justify-center'>
-      {events.map((e) => (
-        <Link key={e.id} href={`/events/${e.id}`}>
-          <h2>{e.name}</h2>
+    <main>
+      <div className='flex flex-col items-center justify-center bg-green-300'>
+        <Link href='http://bealcoholic.com'>
+          <h1 className='text-4xl font-bold m-3'>BeAlcoholic</h1>
         </Link>
-      ))}
+      </div>
+      <div className='flex flex-col items-end justify-end mr-20 mt-10'>
+        <Button variant='default' asChild>
+          <Link href='/events/new'>
+            <Plus className='size-5 -ml-1' /> Új Esemény
+          </Link>
+        </Button>
+      </div>
+
+      <div className='flex flex-col items-center justify-center'>
+        <div className='w-[61rem]'>
+          {events.map((event) => (
+            <div key={event.id} className='my-4'>
+              {/* <Link href={`/events/${event.id}`}> */}
```

Comment:
why is this commented out?

Reply:
ohh I see, the other buttons don't work. I'll show you how it can be fixed
spoiler: https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation

Reply:
Thank you SO MUCH! I spent hours trying to find the answer without any success.